torch/distributed/elastic/multiprocessing/tail_log.py

```diff
@@ -62,7 +62,7 @@ class TailLog:
     ::
 
      log_files = {0: "/tmp/0_stdout.log", 1: "/tmp/1_stdout.log"}
-     tailer = TailLog("trainer", log_files, sys.stdout).start()
+     tailer = TailLog("trainer", log_files, sys.stdout, {}).start()
```

Comment:
you can leave this documentation as is by making the ctor of `TailLog` BC by defaulting `log_line_headers` to `None` , then per @d4l3k 's point below, add an extra paragraph to this docstring on how to customize the log line prefix.